packages/components/pie-link/src/link.scss

```diff
@@ -65,11 +70,9 @@
 
     &[underline='default'] {
         /* Same as default styles */
-
-        @include link-interactive-states('default');
     }
 
-    &[underline='reversed'] {
+    &[underline='reversed'][isStandalone] {
```

Comment:
In the changelog it says "When the `isStandalone` prop is `false`, underline can only be default and not reversed."

But this looks like this applies the styles when underline is `reversed` and `isStandalone` is true?

Reply:
aha sorry bad wording, I will update the changelog. 
but basically, the reverse type must be used only if the `isStandalone` is true. (only if the link is a block element as reversed + inline will be confusing for users).